.github/actions/unit-test/action.yml

```diff
@@ -41,7 +43,7 @@ runs:
       shell: bash
       env:
         DOCKER_REGISTRY: ${{ inputs.docker-registry }}
-        NODE_OPTIONS: --max-old-space-size=8192
```

Comment:
Could this cause failures? We added this to fix large tests running out of memory